src/targets/gpu/target.cpp

```diff
@@ -218,6 +219,10 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         dead_code_elimination{},
         fuse_ops{&ctx, options.fast_math},
         dead_code_elimination{},
+#if MIGRAPHX_USE_HIPBLASLT
```

Comment:
Question: do we want this always on if we can compile it in then not enabled similar to how we use MLIR?

Reply:
This is on by default, we use the env flags to control enabling hipblaslt.

Reply:
All good.